AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+| [Subscription metadata](Readme.md#subscription-metadata)| Subscription metadata  contains each subscription details like subscription id, tenant id, state etc. |
+| [Assessment data](Readme.md#assessment-data)| Assessment Details  contains the evaluated assessment details with the appropriate status reason.|
+| [Policy State data](Readme.md#policy-state-data) | Policy State  contains the evaluated Policy State details with the assigned scope and it's state.|
+| [Role definition data](Readme.md#role-definition-data)| Role Definition Details  contains both custom and built in roles information along with assigned scope.|
+| [Secure Score Control data](Readme.md#secure-score-control-data) | Secure Score Control  contains the secure score evaluated control details with the current score and healthy and unhealty resource count.|
+| [Secure Score data](Readme.md#secure-score-data) | Secure score  contains the evaluated secure score details current and maximum score. |
+___
+
+### **RBAC data**
+
+|Field Name|Field Type|Description|Sample Value| 
+|--|--|--|--|
+|JobId|int|Job ID for scan (YYYYMMDD).|20230414|
+|RoleName|string|The role assignment name.|Reader|
+|RoleId|string|The role assignment ID.|xx0x0xx0-0xx0-0000-0x00-000x0x00000x|
+|AccountType|string|The role assignment account type.|ServicePrincipal|
+|UserName|string|The role assignment user name.|xx0000x0-00x0-000x-000x-x0x00x00xxxx|
+|UserNameMApped|string|User name mapped to role assignment user name.|xx0000x0-00x0-000x-000x-x0x00x00xxxx|
+|PrincipalName|string|The principal name.|user_alias@microsoft.com|
+|UserMail|string|The mail of user.|user_alias@microsoft.com|
+|DisplayName|string|The role assignment name.|Username|
+|Scope|string|The role assignment scope.|/providers/Microsoft.Management/managementGroups/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|NameId|string|The subscription ID of role assignment.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|RBACSource|string|Scope of role assignment.|subscription|
+|RoleDefinitionId|string|Role definition ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/providers/Microsoft.Authorization/roleDefinitions/Role_Def_Id|
+|IsPIMEligible|bool|Indicates whether role assignment is of type PIM.|false|
+|RBACAPISource|string|Indicates source of the RBAC API.|ARMAPI|
+|IsInherited|bool|Indicates whether the role assignment is inherited or assigned at current scope.|false|
+|CustomField1|string|This field contains custom data.|This field contains custom data.|
+|CustomField2|string|This field contains custom data.|This field contains custom data.|
+|CustomField3|string|This field contains custom data.|This field contains custom data.|
+|CustomField4|string|This field contains custom data.|This field contains custom data.|
+|CustomField5|string|This field contains custom data.|This field contains custom data.|
+___
+
+### **Baseline control data**
+
+|Field Name|Field Type|Description|Sample Value|
+|--|--|--|--|
+|OrgTenantId|string|The tenant ID.|null|
```

Comment:
We need to figure this out. A sample value of null doesn't help.

Reply:
Just adds to the confusion

Reply:
Same comment for all such fields which have a "null" sample

Reply:
But there is null value in whole data for these fields. So we cannot publish any random values.

Reply:
Again, if that's the case then might as well not mention this field. Simply mentioning a field name and a sample value of null doesn't look good at all